### PR TITLE
organized objects.yml and started quests and canoes

### PR DESCRIPTION
**Pull request by jarryd229**
organized objects.yml in id order and added examines

### CODE REVIEW

File: data/definitions/npcs.yml [hunk 1]
```diff
@@ -72,122 +33,60 @@ chicken:
...
-  crush_def: -15
-  magic_def: -15
-  range_def: -15
+  stab_def: -42
+  att_bonus: -47
+  str_bonus: -53
+  slash_def: -42
+  crush_def: -42
+  magic_def: -42
+  range_def: -42
```
Comment (GregHib) [inline data/definitions/npcs.yml:62]: Any reason why you update this one and not the others?

File: data/definitions/npcs.yml [hunk 2]
```diff
@@ -72,122 +33,60 @@ chicken:
...
-  crush_def: -26
-  magic_def: -26
-  range_def: -26
+  stab_def: -10
+  att_bonus: -10
+  str_bonus: -10
+  slash_def: -10
+  crush_def: -10
+  magic_def: -10
+  range_def: -10
```
Comment (GregHib) [inline data/definitions/npcs.yml]: Sheep aren't attackable so don't need combat stats 🐑

**Comment by GregHib** [comment]
I'm a bit torn on the objects.yml

## Examines
Pro: it's nice to have them all
Con: The objects.yml file is now massive and that'll slow loading times for future modifications, but it'll inevitably get large the more content is added anyway.

## Sorted by id
Pro: easy to find if you know the id of something, nice and orderly/organised
Con: you lose the context information like pairs of doors 

```
prison_door_closed: 79
prison_door_opened: 43046
```

File: data/definitions/npcs.yml (hunk 2 above)
Comment (jarryd229) [reply data/definitions/npcs.yml]: looking at the original it was already there

File: data/definitions/npcs.yml (hunk 1 above)
Comment (jarryd229) [reply data/definitions/npcs.yml:62]: when using the YamlSorter it's done that it switched some things around that one looks to be from duck_grey: id: 6113 or rat: id: 47

File: data/definitions/npcs.yml (hunk 2 above)
Comment (jarryd229) [reply data/definitions/npcs.yml]: i've gone through npcs.yml should be fixed

File: data/definitions/npcs.yml (hunk 1 above)
Comment (jarryd229) [reply data/definitions/npcs.yml:62]: i've gone through npcs.yml should be fixed and made sure bonus are good

**Review by GregHib — APPROVED** [review]
Did a bit of formatting myself and sorted the objects by id + pairs

**Pull request merged**